test/fail_compilation/custom_constraint_errors.d

```diff
@@ -0,0 +1,28 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/custom_constraint_errors.d(27): Error: template `custom_constraint_errors.foo` cannot deduce function from argument types `!()(long)`
+fail_compilation/custom_constraint_errors.d(23):        Candidate is: `foo(T)(T arg)`
+  with `T = long`
+  must satisfy the following constraint:
+`       isSmallEnough!T: long must be smaller or equal than 4 bytes`
```

Comment:
The placement of backticks is just odd to me.

Reply:
I think the backticks are for code highlighting, they were there before, but I could limit them to the `isSmallEnough!T` here, so insert them before the colon

Reply:
The backticks are for highlighting.